app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -0,0 +1,54 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.ProductNotFoundException;
+import com.codesoom.assignment.models.Product;
+import com.codesoom.assignment.models.ProductRepository;
+import org.springframework.stereotype.Service;
+
+import javax.transaction.Transactional;
+import java.util.List;
+
+@Service
+@Transactional
+public class ProductService {
+
+    private final ProductRepository productRepository;
+
+    public ProductService(ProductRepository productRepository) {
+        this.productRepository = productRepository;
+    }
+
+    public List<Product> getProducts() {
+        return productRepository.findAll();
+    }
+
+    public Product getProduct(Long id) {
+        Product product = productRepository
+                .findById(id)
+                .orElseThrow(() -> new ProductNotFoundException(id));
+
+        return product;
+    }
```

Comment:
```suggestion
    public Product getProduct(Long id) {
        return productRepository
                .findById(id)
                .orElseThrow(() -> new ProductNotFoundException(id));
    }
```

이렇게 바로 반환해줘도 좋을 것 같습니다